package.json

```diff
@@ -119,6 +121,8 @@
     "react": "^16.8.6",
     "react-dom": "^16.13.0",
     "react-is": "^16.13.0",
+    "@testing-library/jest-dom": "^5.11.5",
+    "@testing-library/react": "^11.1.1",
```

Comment:
I know, the dependencies are not needed for the `tsdx` package itself. But I get an error in the `index.test.js` files if the testing library dependencies are not installed. Shall I just suppress those errors?

Reply:
I added it because VSCode displayed an error (`Cannot find module '@testing-library/react' or its corresponding type declarations.`). But you are of course right, they are not needed.